binding/java/test/ai/picovoice/rhino/RhinoTest.java

```diff
@@ -56,25 +92,26 @@ void tearDown() {
     }
 
     @Test
-    void getVersion() {
+    void getVersion() throws RhinoException{
+        buildDefaultTestRhino();
```

Comment:
delete Rhino after each test too. It wasn't there in the original tests, but it should have been since that tests the delete as well.

Reply:
I think there is a function named 'tearDown' that does that